dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java

```diff
@@ -311,7 +311,9 @@ boolean isFromLibProvidedPath(String entryName) {
     protected String hash(JarFile jf, ZipEntry entry) throws IOException, NoSuchAlgorithmException {
         InputStream eis = jf.getInputStream(entry);
         // sha-1 is used temporarily while doing checkpoint/restore for Spring Boot applications until sha-256 is made available by JVM.
-        MessageDigest digest = getDigest(CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA_256.toLowerCase(), CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA_1.toLowerCase());
+        String[] allowedAlgorithms = CryptoUtils.isFips140_3EnabledWithBetaGuard() ? new String[] { CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA_256.toLowerCase() } : new String[] { CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA_256.toLowerCase(),
```

Comment:
SHA-1 only used if SHA-256. Maybe this can be fully deleted but at least in the FIPS 140-3 case SHA-256 will be available and we do not need SHA-1 as a fallback. 

Personally, i think we should just update this one globally.

Reply:
@tjwatson -- The SHA-1 fallback will fail on FIPS, so i've removed it only in FIPS cases. But a question for you would be, does this still pose a problem? Could we remove SHA-1 for FIPS and non-FIPS instead?

Reply:
Looking at getDigest, it is redundant without checking a list of algorithms. Instead of updating the method i removed it entirely and replaced it with 
```
MessageDigest digest = MessageDigest.getInstance(CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA_256);
```